src/views/v2/list.php

```diff
@@ -1,17 +1,36 @@
-<main>
-	<header>
-		<h3>List View (WIP)</h3>
-	</header>
-	<main>
-		<?php foreach ( $events as $event ) : ?>
-			<article>
-				<header>
-					<h4><?php echo $event->post_title ?></h4>
-					<main>
-						<?php echo $event->post_content ?>
-					</main>
-				</header>
-			</article>
-		<?php endforeach; ?>
-	</main>
-</main>
\ No newline at end of file
+<?php
+/**
+ * View: List
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/list.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+
+$events_label_plural = tribe_get_event_label_plural();
+
+$this->template( 'events-bar' );
+
+$this->template( 'top-bar' );
```

Comment:
maybe call it header? To avoid having `filter-bar`, `events-bar` and `top-bar`

Reply:
I followed the terminology used by the designers -> https://moderntribe.invisionapp.com/d/main/#/console/17163687/359236873/preview

Maybe we can discuss a bit about it?